cumulusci/cli/tests/test_org.py

```diff
@@ -90,17 +90,23 @@ def test_org_browser_url_only(self, browser_open, click_echo):
         click_echo.assert_called_once_with(start_url)
         org_config.save.assert_called_once_with()
 
-    @mock.patch("cumulusci.cli.org.CaptureSalesforceOAuth")
+    @mock.patch("cumulusci.cli.org.OAuth2Client")
     @responses.activate
-    def test_org_connect(self, oauth):
-        oauth.return_value = mock.Mock(
-            return_value={
-                "instance_url": "https://instance",
-                "access_token": "BOGUS",
-                "id": "OODxxxxxxxxxxxx/user",
+    def test_org_connect(self, oauth2client):
+        client_instance = mock.Mock()
+        client_instance.auth_code_flow.return_value = {
+            "instance_url": "https://instance",
+            "access_token": "BOGUS",
+            "id": "OODxxxxxxxxxxxx/user",
+        }
+        oauth2client.return_value = client_instance
+        runtime = mock.Mock()
+        runtime.keychain.get_service.return_value = mock.Mock(
```

Comment:
Why is this now needed? org_connect doesn't call get_service from what I can see

Reply:
org_connect calls get_service on line 144 to fetch the connected app so that we can get values for client_id and client_secret.

Reply:
Ah right -- and before it got a mock and that was fine, but now we need something that can be validated by pydantic.